librad/src/meta/entity_test.rs

```diff
@@ -325,16 +327,16 @@ async fn test_project_update() {
         })
     ));
 
-    // History with single valid user is valid
     history
         .revisions
         .last_mut()
         .unwrap()
         .sign(&K1, &Signatory::OwnedKey, &EMPTY_RESOLVER)
         .await
         .unwrap();
-    assert!(matches!(history.check().await, Ok(())));
-    assert_eq!(history.status(), Some(&VerificationStatus::Verified));
+
+    // History with single valid user is valid
+    assert!(matches!(history.check().await, Ok(_)));
```

Comment:
For future work, I'd prefer we use `assert_eq` and the expected value. These kinds of checks can be too permissive, in my opinion. For example, I witnessed the weird behaviour [above](https://github.com/radicle-dev/radicle-link/pull/147/files#diff-e8128eef3fbc5e6398034e0c89623737R300).

Also, `assert_eq` gives better information about the differences, instead of just having a `false` :shrug: